src/content/docs/guides/prerequisites/index.mdx

```diff
@@ -270,15 +270,15 @@ export JAVA_HOME="/Applications/Android Studio.app/Contents/jbr/Contents/Home"
 
 ```sh
 export ANDROID_HOME="$HOME/Android/Sdk"
-export NDK_HOME="$ANDROID_HOME/ndk/25.0.8775105"
+export NDK_HOME="$ANDROID_HOME/ndk/$(ls -1 $ANDROID_HOME/ndk)"
```

Comment:
```suggestion
export NDK_HOME="$ANDROID_HOME/ndk/$(ls -1 $ANDROID_HOME/ndk)" // 23.1.2.3.4 or higher
```

hi, thanks for this great idea. After a [quick discussion on discord](https://discord.com/channels/616186924390023171/662624589037436928/1232473143692623938) the conclusion is that is best to at least say the minimum supported version. Could be added as a comment or as a note. What do you think?

Reply:
I think the minimum version requirement should be stated during the installation step "3. Use the SDK Manager in Android Studio to install the following: ... - NDK (Side by side)", because otherwise, they would have to backtrack.

My suggestion would be to add a paragraph below that:

> Selecting "Show Package Details" in the SDK Manager enables the installation of older package versions. Please only select other than the current packages if you know what you are doing.